xds/src/main/java/io/grpc/xds/LocalityStore.java

```diff
@@ -91,9 +90,9 @@
     private final LoadStatsStore loadStatsStore;
     private final OrcaPerRequestUtil orcaPerRequestUtil;
     private final OrcaOobUtil orcaOobUtil;
-
-    private final Map<XdsLocality, LocalityLbInfo> localityMap = new LinkedHashMap<>();
-    private Map<XdsLocality, LocalityInfo> edsResponsLocalityInfo = ImmutableMap.of();
+    private final PriorityManager priorityManager = new PriorityManager();
+    private final Map<XdsLocality, LocalityLbInfo> localityMap = new HashMap<>();
+    private Set<XdsLocality> loadStatsStoreLocalities = ImmutableSet.of();
```

Comment:
This name is wired tho. `effectiveLocalities`, `currentLocalities` or just `localities` sounds bette. And put a comment 
```java
// Localities to be load balanced instructed by traffic director.
```
would make this much more elegant.

Reply:
Done. `effectiveLocalities` is misleading, there's multiple kinds of "effective" localities for stats store and priority manager. The latter two names sounds better.